src/components/labels/Label.stories.jsx

```diff
@@ -804,14 +428,10 @@ export const LongText = args => {
   );
 };
 
-// eslint-disable-next-line
-const onCloseMock = () => {};
-
-LongText.args = {
-  ...Default.args,
+TrimmedText.args = {
   title: 'Long long long text',
   children: 'Very very long label',
   onClose: {control: onCloseMock},
+  color: LABEL_COLORS_SET.BLUE,
+  onClick: onClickMock,
```

Comment:
why do we need onClick?

Reply:
Actually it's unnecessary, adde by mistake. Thanks for catch